lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2472,8 +2477,113 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
             tx `shouldBe` Left ErrBalanceTxMaxSizeLimitExceeded
 
     describe "when passed unresolved inputs" $ do
-        xit "may fail"
-            $ property prop_balanceTransactionUnresolvedInputs
+        it "fails with some error" $ do
```

Comment:
Note: this applies to a `PartialTx` with one or more inputs that don't appear in the wallet's UTxO set and also don't appear in the explicit list of inputs passed to `balanceTx`.